### ISSUE TITLE
Can't disable "import-order"

### ISSUE THREAD

**Issue by ChaoticMind** [dreadatour/Flake8Lint#82]
"import-order" is supposed to be disabled by default, but not only is it enabled for me, but I also can't seem to be able to disable it. 

In a simple file titled test.py:
```python
#!/usr/bin/env python
import sys
import datetime

print(datetime.datetime.now())
sys.exit()
```
I get an I100 error (imports in the wrong order error) even though:
- <kbd>Preferences</kbd>-><kbd>Package Settings</kbd>-><kbd>Python Flake8 Lint</kbd>-><kbd>Settings - Default</kbd> has ```"import-order": false,``` (which is correctly there by default)
- <kbd>Preferences</kbd>-><kbd>Package Settings</kbd>-><kbd>Python Flake8 Lint</kbd>-><kbd>Settings - User</kbd> is empty, and I tried explicitly adding ```{"import-order": false,}``` to no avail.
- The file is not in a project
- No global flake8 config is present (```.config/flake8``` doesn't exist)
- No ```tox.ini``` file is present (nor ```setup.cfg``` nor .pep8)

Additionally:
- Adding ```"pydocstyle": false,``` to the user settings file works correctly (ie disables the pydocstyle linting), meaning the user settings file is being read properly
- Toggling ```"import-order-style":``` to ```"google",``` instead of ```"cryptography,"``` in the "default" settings file successfully changes the linting, meaning the "default" config file is also being read.

Any ideas? Is it just me?

**Comment by wrhansen** [comment]
I'm getting this issue too. Can't seem to disable the import-order linter through the settings.

**Comment by iffy** [comment]
I'm getting this too, and it's driving me nuts.
<img width="898" alt="screen shot 2016-02-04 at 12 41 26 pm" src="https://cloud.githubusercontent.com/assets/161727/12827402/cda0620e-cb3c-11e5-881f-9d067f92bdb3.png">

**Comment by dreadatour** [comment]
Sorry, guys, my bad. Fixed, please, check.

**Comment by ChaoticMind** [comment]
Thanks!

**Comment by iffy** [comment]
Fixed, thanks

**Comment by bennlich** [comment]
I'm still having this problem--just installed flake8lint today w/ SublimeText2 package manager. Did this get fixed for sublimetext2?
